src/app/meme-card/meme-card.component.scss

```diff
@@ -0,0 +1,57 @@
+.mat-card-image:first-child {
+  margin-top: 0px;
+}
+
+.mat-card-actions {
+  margin: 0px;
+  padding: 20px 0px;
+}
+
+//.meme-card {
+//  min-width: 100px;
+//  max-width: 400px;
+//  max-height: 370px;
+//  min-height: 370px;
+//  display: block;
+//  position: relative;
+//
+//  padding: 10px;
+//  margin: 5px;
+//}
+
+
+.image-frame {
+  overflow: hidden;
+
+  min-width: inherit;
+  max-width: inherit;
+  min-height: 320px;
```

Comment:
`min-height` and `max-height` shouldn't be needed if you set the height explicitly as `height: 320px`. CSS can be funny sometimes though...